assets/src/blocks/ENForm/ENFormInPlaceEdit.js

```diff
@@ -193,34 +204,38 @@ const Signup = ({attributes, setAttributes}) => {
         <div className="title-and-description">
           <RichText
             tagName="h2"
-            value={title}
-            onChange={(title) => { setAttributes({title}) }}
-            placeholder={__('Enter form title', 'planet4-blocks-backend')}
+            value={ title }
+            onChange={ ( titleVal ) => {
+              setAttributes( { titleVal } );
```

Comment:
I'm pretty sure that here it should be `setAttributes( { title: titleVal } );`

Reply:
Done.